test/e2e/workflow/deployment_test.go

```diff
@@ -133,6 +133,57 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			components = append(components, MacvtapComponent)
 			testConfigUpdate(gvk, configSpec, components)
 		})
+		Context("and PlacementConfiguration is deployed", func() {
+			BeforeEach(func() {
+				currentPlacementConfiguration := &cnao.PlacementConfiguration{
+					Workloads: &cnao.Placement{ NodeSelector: map[string]string{
+						"node-role.kubernetes.io/worker": ""},
+					},
+					Infra: &cnao.Placement{ NodeSelector: map[string]string{
+						"node-role.kubernetes.io/master": ""},
+					},
+				}
+				configSpec := cnao.NetworkAddonsConfigSpec{
+					PlacementConfiguration: currentPlacementConfiguration,
+				}
+				components := []Component{}
+
+				By("Deploying PlacementConfiguration")
+				testConfigCreate(gvk, configSpec, components)
+
+				By("Checking that deployed components remain up and running")
+				CheckConfigCondition(gvk, ConditionAvailable, ConditionTrue, CheckImmediately, time.Minute)
+
+				By("Checking PlacementConfiguration was updated")
+				config := GetConfig(gvk)
+				Expect(ConvertToConfigV1(config).Spec.PlacementConfiguration).To(Equal(currentPlacementConfiguration), "Should succeed deploying PlacementConfiguration on non pre-existing config")
+			})
+
+			It("should be able to update PlacementConfigurations", func() {
+				updatedPlacementConfiguration := &cnao.PlacementConfiguration{
+					Workloads: &cnao.Placement{ NodeSelector: map[string]string{
+						"kubernetes.io/hostname": "node02"},
```

Comment:
What if somebody runs our tests with a single node? I believe that is the default on `make cluster-up`. Are we even setting 2 nodes on our CI?

Would you please add check raising an error if we don't have enough nodes to https://github.com/kubevirt/cluster-network-addons-operator/blob/master/test/e2e/workflow/main_test.go.

Reply:
I agree, I can simply change it to `node01` too, the only thing that matters for this test is that the change sticks

Reply:
Sounds good enough